middleware-catalog-module/catalog-model/src/main/java/co/tpg/catalog/model/Subject.java

```diff
@@ -0,0 +1,68 @@
+package co.tpg.catalog.model;
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBAttribute;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBHashKey;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBIgnore;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTable;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * Response class for Subject entity
+ * 
+ * @author maxx
+ * @since 2019-10-03
+ */
+
+@Builder
+@Getter
+@Setter
+@ToString
+@DynamoDBTable(tableName = "Subject")
+public class Subject extends AbstractModel<String> {
```

Comment:
Add new lombok annotations in the top of the class:
@NoArgsContructor
@AllArgsConstructor